### ISSUE TITLE
Ability to add text outside regions

### ISSUE THREAD

**Issue by AiltonSilvaSC** [JasonLautzenheiser/trizbort#493]
![image](https://user-images.githubusercontent.com/19511091/71841183-9372e280-309d-11ea-9051-40b45ac09fb4.png)

**Comment by JasonLautzenheiser** [comment]
I could (and may) add something for this....however you can mimic this by making a room with no borders.

**Comment by AiltonSilvaSC** [comment]
The bad thing of making this, is on export

**Comment by JasonLautzenheiser** [comment]
Yes that's true.....the export would write code for those "rooms".....  I think having a distinct text object is the way to go when that get implemented.

Might actually be useful as well to have an attribute on the room to export or not.

**Comment by AiltonSilvaSC** [comment]
i don't think so, if you create a room, you will want to export
